s4e-web/src/app/views/map-view/map-view.component.ts

```diff
@@ -83,7 +83,10 @@ export class MapViewComponent implements OnInit, OnDestroy {
   public showProductDescription$: Observable<boolean> = this.mapQuery.selectShowProductDescription();
   public selectedLocation$: Observable<LocationSearchResult | null>;
   public overlays$: Observable<UIOverlay[]> = this.overlayQuery.selectVisibleAsUIOverlays();
-  public userIsZK$: Observable<boolean> = this.sessionQuery.selectMemberZK();
+  public userIsAuthorizedForAdditionalFunctionalities$: Observable<boolean> = forkJoin([
+    this.sessionQuery.selectPakMember(),
+    this.sessionQuery.selectMemberZK()
+  ]).pipe(mapAnyTrue());
```

Comment:
`forkJoin` is not appropriate here - use something like `combineLatest`.
`forkJoin` emits value when all it's observable finishes, which will not occur in this case.

(`forkJoin` is good for combining http requests which completes)